src/routes/mods/+page.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+	import Breadcrumbs from "$lib/components/Breadcrumbs.svelte";
+	import { page } from "$app/stores";
+	let showSignInButton = false;
+	import Signin from "$lib/components/Signin.svelte";
```

Comment:
put imports at the top like
```suggestion
	import Signin from "$lib/components/Signin.svelte";
    let showSignInButton = false;
```